project/plugins.sbt

```diff
@@ -8,6 +8,5 @@ resolvers += Resolver.typesafeRepo("releases")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.24.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.5.0")
 addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.5")
-addSbtPlugin("com.typesafe.sbt"  % "sbt-gzip"           % "1.0.2")
+addSbtPlugin("com.github.sbt"    % "sbt-gzip"           % "2.0.0")
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"       % "2.4.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-test-report"    % "0.26.0")
```

Comment:
not needed now that we moved browser tests to other repo, have also removed from build-jobs